Qausim/epower_react_frontend
# Fetch Posts from API

#### What does this PR do?
Fetches posts from the API to serve

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm start` and visit `localhost:3000` dynamic API data should be served

#### Any background context you want to provide?
Data should come from the API
